packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -282,6 +287,18 @@ abstract class ServerlessClusterBase extends Resource implements IServerlessClus
    */
   public abstract readonly connections: ec2.Connections;
 
+  /**
+   * The cluster arn.
```

Comment:
```suggestion
   * ARN of the cluster
```

rationale: consistency with the naming of the property on line 27 (fine if you want to change that one around instead)